### PR TITLE
backend/refactor:#148 moved external integration to nammayatri

### PR DESCRIPTION
**Pull request by Dwibedi-10**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring
- [ ] Dependency updates

## Description
Moved external integration from shared kernel to nammayatri repo.

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [x] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/Select.hs [hunk 1]
```diff
@@ -18,13 +18,14 @@ import qualified Beckn.Types.Core.Taxi.API.Select as Select
 import qualified Beckn.Types.Core.Taxi.Common.ItemCode as OS
 import qualified Domain.Action.Beckn.Select as DSelect
 import Domain.Types.Vehicle.Variant
-import qualified Kernel.External.Maps as Maps
+-- import qualified Lib.Maps as Maps
```
Comment (roman-bodavskiy) [inline Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/Select.hs:21]: redundant comment

File: Backend/app/provider-platform/static-offer-driver-app/Main/src/Tools/Maps.hs [hunk 2]
```diff
@@ -28,7 +28,15 @@ where
 import Domain.Types.Merchant
 import qualified Domain.Types.Merchant.MerchantServiceConfig as DOSC
 import Domain.Types.Merchant.MerchantServiceUsageConfig (MerchantServiceUsageConfig)
-import Kernel.External.Maps as Reexport hiding
+-- import qualified Lib.Maps as Maps
```
Comment (roman-bodavskiy) [inline Backend/app/provider-platform/static-offer-driver-app/Main/src/Tools/Maps.hs:31]: redundant comment

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Tools/Verification.hs [hunk 3]
```diff
@@ -25,18 +25,21 @@ where
 
 import qualified Domain.Types.Merchant as DM
 import qualified Domain.Types.Merchant.MerchantServiceConfig as DMSC
-import Kernel.External.Verification as Reexport hiding
+-- getTask,
```
Comment (roman-bodavskiy) [inline Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Tools/Verification.hs:28]: redundant comments

File: Backend/lib/external-services/src/Lib/Maps/Google/MapsClient.hs [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+  distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+
+  FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero
+
+  General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Lib.Maps.Google.MapsClient
+  ( module GoogleMaps,
```
Comment (roman-bodavskiy) [inline Backend/lib/external-services/src/Lib/Maps/Google/MapsClient.hs:16]: Maybe `Services.Maps.Google.MapsClient `? @DonDokidon @hkmangla

File: Backend/lib/external-services/src/Lib/Maps/MMI/MapsClient/Types.hs
Comment (pavelzarubin) [inline Backend/lib/external-services/src/Lib/Maps/MMI/MapsClient/Types.hs:1]: I think you can delete the MapsClient folder and move the Types module to the MMI folder, as there is nothing in MapsClient except the Types module.

File: Backend/app/dashboard/CommonAPIs/src/Dashboard/Common/Message.hs [hunk 5]
```diff
@@ -27,12 +27,12 @@ import Data.ByteString.Lazy as BSL
 import Data.OpenApi hiding (description, name, password, summary, title, url)
 import Data.Text as T
 import Data.Text.Encoding as DT
-import Kernel.External.Types (Language)
```
Comment (DonDokidon) [inline Backend/app/dashboard/CommonAPIs/src/Dashboard/Common/Message.hs:30]: Do not change pathes, save original structure

**Review by DonDokidon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**